pytest.ini

```diff
@@ -22,3 +22,12 @@ filterwarnings =
     ignore:Using the slower implementation::cupy
     ignore:Jitify is performing a one-time only warm-up::cupy
     ignore:.*scipy.misc.*:DeprecationWarning
+
+markers =
+    slow: Tests that are very slow
+    xslow: mark test as extremely slow (not run unless explicitly requested)
+    xfail_on_32bit: mark test as failing on 32-bit platforms
+    array_api_backends: test iterates on all array API backends
+    array_api_compatible: test is compatible with array API
+    skip_xp_backends(backends, reason=None, np_only=False, cpu_only=False, exceptions=None): mark the desired skip configuration for the `skip_xp_backends` fixture
+    xfail_xp_backends(backends, reason=None, np_only=False, cpu_only=False, exceptions=None): mark the desired xfail configuration for the `xfail_xp_backends` fixture
```

Comment:
pytest would complain with the previous method of adding custom markers to `conftest.py`, as there were references to `pytest.mark.array_api_backends` in `conftest.py` before `conftest.py::pytest_configure()` could be executed.